lighthouse-core/test/report/html/renderer/report-renderer-test.js

```diff
@@ -201,6 +201,51 @@ describe('ReportRenderer', () => {
       assert.ok(/\dx/.test(descriptionsTxt), 'should have added CPU');
       assert.ok(descriptionsTxt.includes(sampleResults.userAgent), 'user agent populated');
     });
+
+    describe('axe-core', () => {
+      let axe;
+
+      beforeAll(() =>{
+        // Needed by axe-core
+        // https://github.com/dequelabs/axe-core/blob/581c441c/doc/examples/jsdom/test/a11y.js#L24
+        global.window = global.self;
+        global.Node = global.self.Node;
+        global.Element = global.self.Element;
+
+        axe = require('axe-core');
+      });
+
+      afterAll(() => {
+        global.window = undefined;
+        global.Node = undefined;
+        global.Element = undefined;
+      });
+
+      it('renders without axe violations', (done) => {
+        const output = renderer.renderReport(sampleResults, renderer._dom._document.body);
+
+        const config = {
+          rules: {
+            // Reports may have duplicate ids
+            // https://github.com/GoogleChrome/lighthouse/pull/5058#discussion_r184791923
+            'duplicate-id': {enabled: false},
```

Comment:
this would of saved us from #9151 :)

Reply:
We can open an issue after this to get rid of the duplicate ids, I think they are used for JS hooks, but it does mean we lose the ability to scroll into view with #first-audit-id when reports are viewed in a browser.